samples/3_map_state_simple/src/app/esri-map/esri-map.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, OnInit, ViewChild, ElementRef } from '@angular/core';
+import { loadModules } from 'esri-loader';
+
+import { MapStateService } from '../services/map-state.service';
+
+@Component({
+  selector: 'app-esri-map',
+  templateUrl: './esri-map.component.html',
+  styleUrls: ['./esri-map.component.css']
+})
+
+export class EsriMapComponent implements OnInit {
+
+  public mapView: __esri.MapView;
```

Comment:
Minor stylist recommendation is to use `import esri = __esri;` and that should help clean up all the double underscore references. I made the same update to my repo https://github.com/andygup/angular-cli-esri/blob/master/esri-map.component.ts#L3